engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-base-subscriptions.controller.js

```diff
@@ -32,7 +32,8 @@ angular.module('Bastion.content-hosts').controller('ContentHostBaseSubscriptions
             'id':                       $scope.$stateParams.contentHostId,
             'organization_id':          CurrentOrganization,
             'sort_by':                  'name',
-            'sort_order':               'ASC'
+            'sort_order':               'ASC',
+            'match_system':             true
```

Comment:
Unfortunately in the new UI we lost a SAM-1 feature that allowed choosing what to display: All, match system, match installed software, and no overlap with current. Showing all is wrong and match system is safest all around. To further complicate things, errors attaching subscriptions are not making it from dynflow up to the UI so the reason why a subscription failed to attach would be unknown. Specifying 'match system' seems safest all around at this point.